src/lab13.py

```diff
@@ -306,64 +306,25 @@ def parse_transform(transform_str):
 
 @wbetools.patch(CSSParser)
 class CSSParser:
-    def __init__(self, s):
-        self.s = s
-        self.i = 0
-
-    def whitespace(self):
-        while self.i < len(self.s) and self.s[self.i].isspace():
-            self.i += 1
-
-    def literal(self, literal):
-        if not (self.i < len(self.s) and self.s[self.i] == literal):
-            raise Exception("Parsing error")
-        self.i += 1
-
-    def word(self):
```

Comment:
I think it's dangerous to remove this version of word right now. Maybe it was added
to make the parser not crash on real pages, I'm not sure. Since it isn't in the book I don't
think it is urgent to remove it.

Reply:
Ok, that's fair, I can put the `word` function back in. I *am* pretty sure it isn't necessary (we only call `word` for property *names* and media-query property names) but having it in won't affect the outline so doesn't matter much for the book.